charts/portal/README.md

```diff
@@ -588,9 +607,10 @@ $ kubectl delete pvc data-rabbitmq-0|1|2
 ```
 $ kubectl scale statefulset rabbitmq --replicas=1|3
 ```
+Once the rabbitmq is running make a note of its credentials as specified in above Install Chart section 
 
 #### Your Kubernetes nodes failed or RabbitMQ crashed.
-If the RabbitMQ nodes are stopped or removed out of order, there is a chance that it won't be restored correctly.
+If the RabbitMQ cluster is stopped or removed out of order: for instance if the entire cluster loses power, there is a chance that it won't be restored correctly. Or If sync between rabbitmq peers doesn't happen or set of rabbitmq nodes can never be brought online use the 'force boot' option
```

Comment:
If the RabbitMQ cluster is stopped or removed out of order, there is a chance that it won't be restored correctly.

Reply:
updated